analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CastConcreteTypeToInterface.CSharp11.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
```

Comment:
I think we don't need tests for this. The rule subscribes only to cast and `AsIs` expressions and there are no scope-related checks.